src/server.ts

```diff
@@ -89,9 +81,8 @@ class ExecuteAdapters {
             document: DocumentNode,
             rootValue?: any,
             contextValue?: any,
-            variableValues?: {[key: string]: any},
-            operationName?: string,
-    ) => ({
+            variableValues?: { [key: string]: any },
+            operationName?: string,) => ({
       subscribe: (observer) => {
         if (ExecuteAdapters.isASubscriptionRequest(document, operationName)) {
           observer.error(new Error('Subscriptions are not supported'));
```

Comment:
we should replace this with running subscribe, also pass subscribe (optional) to this funtion.
only if subscribe is not defined, throw this error